providers/efa/CMakeLists.txt

```diff
@@ -9,3 +14,8 @@ publish_headers(infiniband
 )
 
 rdma_pkg_config("efa" "libibverbs" "${CMAKE_THREAD_LIBS_INIT}")
+
+if (ENABLE_TRACE AND LTTNGUST_FOUND)
+	target_include_directories(efa PUBLIC ".")
```

Comment:
Huh?

Reply:
This is needed for the tracepoint provider header file: efa_trace.h, LTTng needs each header file to supply the path to the header itself:
```
#define LTTNG_UST_TRACEPOINT_INCLUDE "efa_trace.h"
```
without It, I get the error:
```
fatal error: efa_trace.h: No such file or directory
 #define LTTNG_UST_TRACEPOINT_INCLUDE "efa_trace.h"
```

I tried to change to "efa/efa_trace.h" and "providers/efa/efa_trace.h" and got the same error.

If you look at LTTng-UST github example for CMake: [link](https://github.com/lttng/lttng-ust/blob/master/doc/examples/cmake-multiple-shared-libraries/CMakeLists.txt), you can see that they also used:
```
include_directories(.)
```